storages/response_picker.go

```diff
@@ -164,8 +170,37 @@ func (drp *deleteResponsePicker) pullResponses(out chan<- BackendResponse) {
 	close(drp.syncLogReady)
 }
 
+
+func pullResponsesWatchdog(drp *baseDeleteResponsePicker, out chan<- BackendResponse) {
+	shouldSend := false
+	emptyResponse := BackendResponse{}
```

Comment:
Move it to package variable, there is no need for creating that on each request. GC will be grateful